hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAzureBlobFileSystemCheckAccess.java

```diff
@@ -184,14 +185,17 @@ public void testCheckAccessForAccountWithoutNS() throws Exception {
     checkIfConfigIsSet(FS_AZURE_BLOB_FS_CHECKACCESS_TEST_CLIENT_SECRET);
     checkIfConfigIsSet(FS_AZURE_BLOB_FS_CHECKACCESS_TEST_USER_GUID);
 
-    setTestUserFs();
-
     //  When the driver does not know if the account is HNS enabled or not it
     //  makes a server call and fails
-    intercept(AccessControlException.class,
-        "\"This request is not authorized to perform this operation using "
-            + "this permission.\", 403",
-        () -> testUserFs.access(new Path("/"), FsAction.READ));
+    intercept(Exception.class,
+            "\"This request is not authorized to perform this operation using "
+                    + "this permission.\", 403", () ->
+    setTestUserFs());
```

Comment:
we can remove `setTestUserFs`?

Lets clone the configuration object. reason being in parallel test run it will reflect change in other tests as well.

Reply:
Added as separate function.